analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "workspace",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "workspaces": [
+      "packages/*"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "private": true,
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```

Comment:
License should be Apache-2.0 to match our own license.
Comments on this file apply to the two `package.json`s below as well.

Reply:
Done.